src/transaction/requests.rs

```diff
@@ -315,11 +330,14 @@ impl KvRequest for kvrpcpb::PrewriteRequest {
         store_stream_for_keys(mutations, pd_client)
     }
 
-    fn map_result(_: Self::RpcResponse) -> Self::Result {}
+    fn map_result(response: Self::RpcResponse) -> Self::Result {
+        response
+    }
 
     async fn reduce(results: BoxStream<'static, Result<Self::Result>>) -> Result<Self::Result> {
         results
-            .try_for_each_concurrent(None, |_| future::ready(Ok(())))
+            .into_future()
```

Comment:
I don't get it why it's changed to `into_future`. We may have multiple prewrite requests.

Reply:
Whoops, yeah that's wrong